dts/x86/quark_d2000_crb.dts

```diff
@@ -0,0 +1,29 @@
+/dts-v1/;
+
+#include "intel_quark_d2000.dtsi"
+
+/ {
+	model = "intel,quark";
+	compatible = "intel,quark";
```

Comment:
Actually this should be something like:

compatible = "intel,quark-d2000-crb", "intel-quark-d2000", "intel,quark";

(I'm not sure if the "intel,quark" actually makes sense).